app/assets/stylesheets/common/base/search.scss

```diff
@@ -157,14 +157,21 @@
     details.advanced-filters,
     details.search-advanced-additional-options {
       margin-top: 1em;
-
       > summary {
         color: var(--tertiary);
         cursor: pointer;
+        padding-top: 0.25em;
+        padding-bottom: 0.25em;
+        &:focus-visible {
```

Comment:
Just noting that `:focus-visible` style won't be present on still-supported iOS 12. But I think we're ok with that.